RiotSharp/ITournamentRiotApi.cs

```diff
@@ -219,8 +373,10 @@ public interface ITournamentRiotApi
         /// <param name="pickType">The pick type.</param>
         /// <param name="mapType">The map type.</param>
         /// /// <returns>Success value.</returns>
-        Task<bool> UpdateTournamentCodeAsync(string tournamentCode, List<long> allowedSummonerIds,
+        [Obsolete]
+        Task<bool> UpdateTournamentCodeV1Async(string tournamentCode, List<long> allowedSummonerIds,
             TournamentSpectatorType? spectatorType, TournamentPickType? pickType, TournamentMapType? mapType);
+
+        #endregion
```

Comment:
Given the timeframe for porting everything to v3, I'd be in favor of making a clean cut removing those `V1` mehods that'll keep the codebase slimmer.